### ISSUE TITLE
consider article covers in native export and import

### ISSUE THREAD

**Issue by bozana** [pkp/pkp-lib#4294]

**Comment by bozana** [comment]
PRs:
ojs-stable-3_1_1: https://github.com/pkp/ojs/pull/2214

**Comment by bozana** [comment]
@asmecher, this PR changes a native export/import element name: issue_covers is renamed into covers, so that both, article and issue, can use the same function i.e. element. OK so?

**Comment by asmecher** [comment]
Yes, that's good, thanks! Did any of the included sample XML need changing?

**Comment by bozana** [comment]
Thanks @asmecher! No need, but I've just added a more comprehensive article import sample.xml...
